### DISCUSSION TITLE
VS 2022 plugin: how to see multiple proposals for the generated code?

### DISCUSSION THREAD

**Discussion by VioletGiraffe** [Exafunction/WindsurfVisualStudio#63] [Q&A]
Hi, I'm using VS 2022 with Codeium (not VS Code). It says "3 proposals generated" in the log, but only shows one continuation option in the main editor that I can either accept with Tab or ignore. How to see the other proposals?

**Comment by fortenforge** [comment]
Not currently possible, but it's a feature we'll add soon.

**Comment by VioletGiraffe** [comment]
Okay, so I was right to think it's just taunting me :) Thanks for the answer!
